src/style.css

```diff
@@ -1,3 +1,77 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap';
+
 body {
   margin: 0;
+  background-color: #e1e3e1;
+  font-family: Roboto, sans-serif, system-ui;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+  align-items: center;
+}
+
+.moyo__logo {
+  height: 40px;
+  margin-left: 50px;
+  padding: 0;
+}
+
+nav {
+  width: auto;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
```

Comment:
Avoid using tag names for styling. In this case, `.nav__list` is sufficient and the `ul` tag name is unnecessary.